src/widgets/content/helpers.rs

```diff
@@ -209,19 +220,40 @@ pub async fn select_task_list(
 
 			match local.get_tasks_from_list(list.id.clone()).await {
 				Ok(response) => {
-					for task in response {
-						guard.push_back(TaskInit::new(
-							task,
-							Some(list.clone()),
-							model.compact,
-						));
+					for task in response
+						.iter()
+						.filter(|task| task.status != Status::Completed)
+						.map(|task| task.to_owned())
+					{
+						guard.push_back(TaskInit::new(task, list.clone()));
 					}
 				},
 				Err(err) => tracing::error!("{err}"),
 			}
 		},
 	}
 
+	model.page_icon = if matches!(model.parent_list, Some(SidebarList::Custom(_)))
+	{
+		"/dev/edfloreshz/Done/icons/scalable/actions/checked.png".into()
+	} else {
+		"/dev/edfloreshz/Done/icons/scalable/actions/empty.png".into()
+	};
+
+	model.page_title =
+		if matches!(model.parent_list, Some(SidebarList::Custom(_))) {
+			fl!("all-done").clone()
+		} else {
+			fl!("list-empty").clone()
+		};
+
+	model.page_subtitle =
+		if matches!(model.parent_list, Some(SidebarList::Custom(_))) {
+			fl!("all-done-instructions").clone()
+		} else {
+			fl!("instructions").clone()
+		};
+
```

Comment:
`matches!(model.parent_list, Some(SidebarList::Custom(_)))` is used multiple times, you could just use a variable to store the result once to make the code more readable.

Reply:
I actually had a method that does that 🤣